### ISSUE TITLE
Clients are stored with invalid predicates

### ISSUE THREAD

**Issue by timbot1789** [codeforpdx/PASS#228]
**Describe the bug**

tl;dr: The data in the 'person' field on users doesn't comply with the schema.org spec.

Currently, the client list is storing clients in the `userlist.ttl` file on client pods with a format like the following:
```
<#http%3A%2F%2Flocalhost%3A3000%2Ftimbot%2Fprofile%2Fcard%23me>
    <http://schema.org/Person> "New Person";
    <http://schema.org/givenName> "New";
    <http://schema.org/familyName> "Person";
    <http://schema.org/identifier> <http://localhost:3000/timbot/profile/card#me>;
    <http://schema.org/URL> <http://localhost:3000/timbot/>.
```
This incorrectly handles the `http://schema.org/Person` predicate. We are using it as if it were a field, but it is actually a _type_. It is meant to describe what values the object holds, not hold a value itself. What we want to say here "this contact, identified by the URI, is a person, and they have the name "New Person"." However, what we're actually saying is "this contact, identified by the URI, persons the text "New Person."

**Requested Fix**

Rewrite the structure of the contacts list so that it is stored with valid RDF data.

A valid structure would be something like:
```
<#http%3A%2F%2Flocalhost%3A3000%2Ftimbot%2Fprofile%2Fcard%23me>
    a  <http://schema.org/Person>;
    <http://schema.org/givenName> "New";
    <http://schema.org/familyName> "Person";
    <http://schema.org/identifier> <http://localhost:3000/timbot/profile/card#me>;
    <http://schema.org/URL> <http://localhost:3000/timbot/>.
```

Note that instead of using `http://schema.org/Person` as a predicate, it uses `a` as the predicate, and `http://schema.org/Person` as the object. The `a` predicate is a reserved term in TTL. You can read more about it [here](https://www.w3.org/TR/turtle/#iri-a)

Another valid approach would be to remove the `Person` entry altogether.

**Additional context**
Most of the behavior about the contacts list is handled in `src/hooks/useContactsList.js`. That would be a good place to start looking for how to fix this

**Comment by timbot1789** [comment]
Very minor issue, and not something that should affect our MVP. Could be a good first issue for someone trying to learn about RDF.

**Comment by Lasiorhine** [comment]
I'm still waiting to be added to the project, but just FYI, I'm going to go ahead and grab this one and start work while I'm waiting.  Cheers.

**Comment by timbot1789** [comment]
@Lasiorhine  Just assigned you. Not sure why you weren't able to assign yourself. We can look into that.

It would be good to connect with @leekahung about this. He just wrote up a profile page which would display some of this information.


Thanks for picking up an issue! Let me know if you have any questions.

**Comment by Lasiorhine** [comment]
Just a quick note (attn: @timbot1789 ) in case this item is discussed at today's meeting:  I had less time than I thought I would last week, but I have a bunch of time blocked out for this in the next three days (i.e., Wednesday, Thursday, and Friday.)   I'll send up a flare if I need help!  Feel free to reach out with any questions.

**Comment by Lasiorhine** [comment]
Example from Tim given at tonight's work session:

{
  <www.schema.org/person>: https://webid.server.com,
  podUrl: https://pod.server.com,
  preferredName: 'Victoria',
  legalName: ""
}

ttl

**Comment by Lasiorhine** [comment]
User.js no longer exists-- I think this bug was fixed when #362 was merged. @timbot1789, can we close this one?

**Comment by xscottxbrownx** [comment]
@timbot1789 what is the status of this?

**Comment by timbot1789** [comment]
@Lasiorhine we talked in a meeting about this, but I forgot to write up a response. This ticket is still valid. It focuses on how the data is stored, not on the front end architecture around the data. Although we've move the code that creates the contacts list to a different spot in the react app, the contacts list itself is still being created with an incorrect structure.
@xscottxbrownx No updates. I cleaned up the description to be easier to understand.
